### ISSUE TITLE
Is there anyway to run PLACER without the protein target?

### ISSUE THREAD

**Issue by ackbar03** [baker-laboratory/PLACER#5]
Hi, 

I saw PLACER can also be used to generate the conformation of the ligand by itself (Figure 2D in the paper https://www.biorxiv.org/content/10.1101/2024.09.25.614868v1). 

How would I do this using the current code? I tried running placer on a PDB containing only the ligand, but it throws errors:

```
    outputs_LIG = placer.run(pl_inp, 10)
  File "/home/ackbar03/proteins/PLACER/zdbg_initialtst/../PLACER.py", line 281, in run
    H = G.subgraph(nx.single_source_shortest_path_length(G, center[0].name, cutoff=cutoff))
  File "/home/ackbar03/miniconda3/envs/placer/lib/python3.10/site-packages/networkx/utils/decorators.py", line 788, in func
    return argmap._lazy_compile(__wrapper)(*args, **kwargs)
  File "<class 'networkx.utils.decorators.argmap'> compilation 4", line 3, in argmap_single_source_shortest_path_length_1
  File "/home/ackbar03/miniconda3/envs/placer/lib/python3.10/site-packages/networkx/utils/backends.py", line 967, in __call__
    return self.orig_func(*args, **kwargs)
  File "/home/ackbar03/miniconda3/envs/placer/lib/python3.10/site-packages/networkx/algorithms/shortest_paths/unweighted.py", line 59, in single_source_shortest_path_length
    raise nx.NodeNotFound(f"Source {source} is not in G")
networkx.exception.NodeNotFound: Source ('A', 1, 'LIG', 'C11') is not in G
```

Thanks!

**Comment by ikalvet** [comment]
The current checkpoint that we have shared was trained for predictions within protein context and has not been validated for this task, so I would be careful with using it for just ligands. We had used an early version of the model (trained just on the CSD) and a development version of inference code to do that particular analysis in the manuscript.

That being said, the error you got seems to point towards a bug in general. I had a look and found that changing from `<` to `<=` on line 383 in `modules/dataloader_pdb.py` like this:
`mask = np.array(list(cropped_dict.values())).cumsum()<=maxatoms`
allows predicting with just the ligand.
I need to investigate how this code change affects the overall behavior of the program, so I'm hesitant to push an update with this change just yet.
